install.php

```diff
@@ -40,6 +40,20 @@ class Com_RedcoreInstallerScript
 	public $installer = null;
 
 	/**
+	 * Extension element name
+	 *
+	 * @var  string
+	 */
+	protected $extensionElement = '';
+
+	/**
+	 * Old version according to manifest
+	 *
+	 * @var  string
+	 */
+	protected $oldVersion = '0.0.0';
```

Comment:
I think this is not proper... because of this line: https://github.com/redCOMPONENT-COM/redCORE/pull/440/files#diff-3b7de2cf163f18aa521c050bb543084fR136
So if you are installing it for the first time, current version (old) will always be lower that the one which is being installed... 
so you will run all sql and php update files on the install, and it is not needed there